modules/ivts/dev.galasa.ivts/dev.galasa.ivts.core/build.gradle

```diff
@@ -0,0 +1,60 @@
+
+// This section tells gradle which gradle plugins to use to build this project.
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'dev.galasa.tests' version '0.40.0'
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+// This section tells gradle where it should look for any dependencies
+repositories {
+    mavenLocal()
+    mavenCentral()
+    // To use the bleeding edge version of galasa's obr plugin, use the development obr
+    maven {
+       url 'https://development.galasa.dev/main/maven-repo/obr'
+    }
+
+}
+
+// Set the variables which will control what the built OSGi bundle will be called
+// and the name it will be published under in the maven repository.
+group = 'dev.galasa.ivts'
+version = '0.0.1-SNAPSHOT'
```

Comment:
If you are building this test as a snapshot, why not set the parent to use snapshot also ?

Reply:
This will be set to 0.40.0 like the rest of Galasa but I will be doing all version related things when the POC works and I'm ready to deliver this PR